server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -1262,9 +1298,72 @@ public void deleteConsumer(
                 returnCerts.add(cert);
             }
         }
+        // we want to insert the content access cert to this list
+        if (!consumer.getOwner().contentAccessMode()
+            .equals(ContentAccessCertServiceAdapter.DEFAULT_CONTENT_ACCESS_MODE)) {
+            try {
+                Certificate cert = contentAccessCertService.getCertificate(consumer);
+                if (cert != null) {
+                    returnCerts.add(cert);
+                }
+            }
+            catch (IOException ioe) {
+                throw new BadRequestException(i18n.tr("Cannot retrieve content access certificate"), ioe);
+            }
+            catch (GeneralSecurityException gse) {
+                throw new BadRequestException(i18n.tr("Cannot retrieve content access certificate"), gse);
+            }
+        }
         return returnCerts;
     }
 
+    @ApiOperation(notes = "Retrieves the body of the Content Access Certificate for the Consumer",
+        value = "getContentAccessBody")
+    @ApiResponses({ @ApiResponse(code = 404, message = ""), @ApiResponse(code = 304, message = "") })
+    @GET
+    @Path("{consumer_uuid}/accessible_content")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getContentAccessBody(
+        @PathParam("consumer_uuid") @Verify(Consumer.class) String consumerUuid,
+        @QueryParam("if-modified-since") Date since) {
```

Comment:
We should use the "If-Modified-Since" header on the request.  That's what the HTTP spec expects: https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html (see section 14.25).  RESTEasy/JAX-RS supports this with the `@HeaderParam` annotation.